modules/site-health/audit-enqueued-assets/load.php

```diff
@@ -25,22 +25,33 @@ function perflab_aea_audit_enqueued_scripts() {
 		global $wp_scripts;
 		$enqueued_scripts = array();
 
-		foreach ( $wp_scripts->queue as $handle ) {
-			$src = $wp_scripts->registered[ $handle ]->src;
-			if ( $src && ! strpos( $src, 'wp-includes' ) ) {
+		foreach ( $wp_scripts->done as $handle ) {
+			$script = $wp_scripts->registered[ $handle ];
+
+			if ( $script->src && ! strpos( $script->src, 'wp-includes' ) ) {
+
+				// Add any extra data (inlined) that was passed with the script.
+				$inline_size = 0;
+				if ( ! empty( $script->extra ) && ! empty( $script->extra['after'] ) ) {
+					foreach ( $script->extra['after'] as $extra ) {
+						$inline_size += ( is_string( $extra ) ) ? mb_strlen( $extra, '8bit' ) : 0;
+					}
+				}
+
+				$path               = perflab_aea_get_path_from_resource_url( $script->src );
 				$enqueued_scripts[] = array(
-					'src'  => $src,
-					'size' => perflab_aea_get_resource_file_size( perflab_aea_get_path_from_resource_url( $src ) ),
+					'src'  => $script->src,
+					'size' => perflab_aea_get_resource_file_size( $path ) + $inline_size,
```

Comment:
Does this size take into account the size on the server / disk - or the transferred / gzipped size? Probably worth noting as the 2 can be different and could skew results.

Reply:
it's the disk/server size. It uses filesize() to get the file size.

Reply:
@manuelRod - that makes sense - just so Im understanding correctly, would that size be more or less the same than transferred / gzipped size?